### DISCUSSION TITLE
yarn android 실행시 에뮬레이터에 화면이 그냥 휴대폰 바탕화면으로만 나옵니다.

### DISCUSSION THREAD

**Discussion by Leesuw** [velopert/dealing-with-react-native#19] [Q&A]
안녕하세요 

책을 구매하고 처음 부터 해보고 있는데 VS Code에서 yarn android 명령어를 입력을 했는데 책처럼 화면이 나오는것이 아니고 계속 스마트폰 바탕화면만 나옵니다.
![리액트 네이티브 질문_1](https://user-images.githubusercontent.com/49264347/151753802-ecb5bb42-4afc-4f4c-b12c-842e58875652.PNG)
![리액트 네이티브 질문_2](https://user-images.githubusercontent.com/49264347/151753808-80dea7fc-946d-4401-9880-e10c6b13b30c.PNG)

**Comment by velopert** [comment]
터미널에 나타난 yarn android 의 결과를 적어주시면 도움을 드릴 수 있습니다.
